polaris.shopify.com/pages/api/v0/tokens/index.tsx

```diff
@@ -62,16 +66,14 @@ const html = `
                 </tr>
             </thead>
             <tbody>
-            ${Object.keys(colorSchemeMap)
-              .map((colorScheme) => {
-                const scheme = colorScheme as Scheme;
+            ${Object.keys(osColorSchemes)
+              .map((scheme) => {
```

Comment:
Same as above, [TBD](). I suggestion either exporting the [`colorSchemes` token groups](https://github.com/Shopify/polaris/blob/5741343151f3ec52842878d8ce607cd8d9fc0053/polaris-tokens/tokens.ts#L49-L52) from `polaris-tokens` or accessing them from `tokens` e.g.
```js
import {tokens} from '@shopify/polaris-tokens'
Object.keys(tokens.colorSchemes).map(...)
```